data-remote/src/main/java/com/teamdontbe/data_remote/api/MyPageUserProfileApiService.kt

```diff
@@ -0,0 +1,17 @@
+package com.teamdontbe.data_remote.api
+
+import com.teamdontbe.data.dto.BaseResponse
+import com.teamdontbe.data.dto.response.ResponseMyPageUserProfileDto
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface MyPageUserProfileApiService {
+    companion object {
+        const val VIEW_MEMBER = "viewmember"
+    }
+
+    @GET("/${LoginApiService.API}/${LoginApiService.V1}/$VIEW_MEMBER/{viewmemberId}")
```

Comment:
```suggestion
    @GET("/${LoginApiService.API}/${LoginApiService.V1}/$VIEW_MEMBER/{viewmemberId}")
```
p1 : LoginApiService -> import 시켜주세욤

Reply:
[4eedf71](https://github.com/TeamDon-tBe/ANDROID/pull/91/commits/4eedf71e7e8dff71d656bc8c42596b772b65d068)